src/explorer/SiteTreeItem.ts

```diff
@@ -166,6 +166,8 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
         } else if (input === dontShowAgainButton) {
             workspace.getConfiguration(constants.extensionPrefix, Uri.file(fsPath)).update(constants.configurationSettings.showBuildDuringDeployPrompt, false);
         }
+
+        telemetryProperties.enableScmInput = input ? input.title : 'Canceled';
```

Comment:
Note that that will send localized titles to telemetry...

Reply:
Good point...